### ISSUE TITLE
Trying to use sklearn Gradient Boosting Classifier

### ISSUE THREAD

**Issue by EliasBrohammer** [adap/flower#1246]
for some day's now I try to implement gradient boosted tree's with flwr. I tried the [example](https://github.com/adap/flower/tree/main/examples/sklearn-logreg-mnist) given. Here you use a LogisticRegression Model for the classification of MNIST, which works in my setup.

This is where my problem arise: Simply replacing the sklear.linear_model.LogisticRegression model with the sklearn.ensemble.GradientBoostingClassifier does not work because I don't know which values I have to initalize, which are equivalent to the LogistiRegression.coef_. Does anyone has an idea?

Thanks in advance

**Comment by gubertoli** [comment]
@EliasBrohammer the GradientBoostingClassifier does not have the coef_ atribute, looking to the sklearn API, probably the [get_params method](https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.GradientBoostingClassifier.html#sklearn.ensemble.GradientBoostingClassifier.get_params) may be useful.

**Comment by EliasBrohammer** [comment]
@gubertoli thanks for the suggestion, but that function returns the (hyper-) parameter like `n_estimators`.
I'm diggin through the documentation and hopefully find something I can work with.

**Comment by zhuygln** [comment]
> @gubertoli thanks for the suggestion, but that function returns the (hyper-) parameter like `n_estimators`. I'm diggin through the documentation and hopefully find something I can work with.

Any progress?

**Comment by EliasBrohammer** [comment]
Unfortunately no progress. I needed it for a project and my time was running low. So I switched to neural networks (tensorflow) with flower, which worked like a charm!
